src/services/consent/services/consentVersionService.js

```diff
@@ -63,7 +63,7 @@ class ConsentVersionService {
 				schoolId,
 				data: consentDocumentData.consentData,
 				filetype: 'pdf',
-				filename: 'Datenschutzerklärung',
+				filename: 'Privacy Policy',
```

Comment:
Our language default is German, so maybe we should keep the file name in German

Reply:
I could change that if that's an issue but this is only for database entry, this field is not used anywhere and the file name user sees is translated on the clients anyway